test/cases/ostree-simplified-installer.sh

```diff
@@ -279,6 +279,24 @@ sudo ostree --repo="$PROD_REPO" remote add --no-gpg-verify edge-stage "$STAGE_RE
 greenprint "🔧 Prepare stage repo network"
 sudo podman network inspect edge >/dev/null 2>&1 || sudo podman network create --driver=bridge --subnet=192.168.200.0/24 --gateway=192.168.200.254 edge
 
+###########################################################
+##
+## Prepare fdo server
+##
+###########################################################
+greenprint "🔧 Prepare fdo manufacturing server"
+sudo git clone https://github.com/runcom/fdo-containers
+cd fdo-containers/ || exit
+sudo ./create-keys.sh
+sudo setenforce 0
```

Comment:
🙈

Reply:
Walsh would scream about that, I need to create a Bz for it

Reply:
ok we just need to relabel with `:z` pushing a commit to fix after tests run